clients/benchmarks/client.cpp

```diff
@@ -353,6 +353,7 @@ struct perf_blas<T,
                 {"tbmv", testing_tbmv<T>},
                 {"tbmv_batched", testing_tbmv_batched<T>},
                 {"tbmv_strided_batched", testing_tbmv_strided_batched<T>},
+                {"trmm", testing_trmm<T>},
```

Comment:
Should be moved down in the build with tensile section?

Reply:
Thanks for catch, moved to build with tensile section